.scalafmt.conf

```diff
@@ -1,47 +1,31 @@
-style = defaultWithAlign
+version = 2.7.5
+preset = default
 maxColumn = 120
 lineEndings = unix
 importSelectors = singleLine
 
-project {
-  git = true
-}
-
-align = none
-
-align {
-  tokens = [ {code = "=>", owner = "Case|Type.Arg.ByName"}, "<-", "->", "%", "%%" ]
-  arrowEnumeratorGenerator = true
-  openParenCallSite = false
-  openParenDefnSite = false
-}
-
-binPack {
-  parentConstructors = true
-}
-
-continuationIndent {
-  callSite = 2
-  defnSite = 2
-}
-
-newlines {
-  penalizeSingleSelectMultiArgList = false
-  sometimesBeforeColonInMethodReturnType = true
-}
-
-rewrite {
-  rules = [RedundantBraces, RedundantParens, AsciiSortImports]
-  redundantBraces {
-    maxLines = 120
-    includeUnitMethods = true
-    stringInterpolation = true
-  }
-}
-
-spaces {
-  inImportCurlyBraces = true
-  beforeContextBoundColon = false
-}
-
-assumeStandardLibraryStripMargin = true
\ No newline at end of file
+project.git = true
+
+align.tokens = [ {code = "=>", owner = "Case|Type.Arg.ByName"}, "<-", "->", "%", "%%" ]
+align.arrowEnumeratorGenerator = true
+align.openParenCallSite = false
+align.openParenDefnSite = false
+
+binPack.parentConstructors = true
+
+continuationIndent.callSite = 2
+continuationIndent.defnSite = 2
+
+newlines.penalizeSingleSelectMultiArgList = false
+newlines.sometimesBeforeColonInMethodReturnType = true
+
+
+rewrite.rules = [RedundantBraces, RedundantParens, AsciiSortImports]
+rewrite.redundantBraces.maxLines = 120
+rewrite.redundantBraces.includeUnitMethods = true
```

Comment:
Is it formatted by **scalafmt** to flat instead of tree structure?

Reply:
No, I rewrote that. 

It looks like it is preferred syntax: https://scalameta.org/scalafmt/docs/configuration.html#other